tests/callbacks/test_gpu_stats_monitor.py

```diff
@@ -116,6 +116,11 @@ def test_gpu_stats_monitor_no_gpu_warning(tmpdir):
 
 
 def test_gpu_stats_monitor_parse_gpu_stats():
```

Comment:
don't we also want to add a test where we `patch.dict()` the `os.environ` with cuda visible devices set? what do you think?

Reply:
I think it will depends on the GPU resources of the CI test container. If the test case is `CUDA_VISIBLE_DEVICES="0"`, it is a trivial case. But if set `CUDA_VISIBLE_DEVICES="1,0"`, the container should have at least 2 GPUs. Otherwise, `torch.cuda` will raise `RuntimeError('CUDA Error: invalid device ordinal')` at device parsing. I am not sure how to force the CI test container use multi-GPU.

Reply:
There is a decorator for that, simply add `@RunIf(min_gpus=2)`